client/clientservice/src/request_service.cpp

```diff
@@ -45,8 +45,23 @@ Status RequestServiceImpl::Send(ServerContext* context, const Request* proto_req
   auto callback = [&](cc::SendResult&& send_result) {
     if (not std::holds_alternative<bft::client::Reply>(send_result)) {
       LOG_INFO(logger_, "Send returned error");
-      if (std::get<int>(send_result) == concord_client_pool::Overloaded)
-        status.set_value(grpc::Status(grpc::StatusCode::RESOURCE_EXHAUSTED, "Overloaded"));
+      switch (std::get<int>(send_result)) {
+        case (concord_client_pool::Overloaded):
+          status.set_value(grpc::Status(grpc::StatusCode::RESOURCE_EXHAUSTED, "ALL CLIENTS OCCUPIED"));
+          break;
+        case (concord_client_pool::Invalid_Argument):
+          status.set_value(grpc::Status(grpc::StatusCode::INVALID_ARGUMENT, "INVALID ARGUMENT SENT"));
```

Comment:
"invalid argument" should be good enough

Reply:
Done